components/blitz/resources/omero/FS.ice

```diff
@@ -398,6 +399,7 @@ module omero {
                                 int blockSize,
                                 bool ignoreSysFiles,
                                 bool ignoreDirEvents,
+                                bool platformCheck,
```

Comment:
Optional parameters are only in Ice 3.5. Not backporting would certainly be an option, but won't help the folks on the lists. I wonder if `export OMERO_PLATFORM_CHECK=false && bin\omero admin restart` would be a workaround?